builtin/v11/check.go

```diff
@@ -327,11 +327,8 @@ func CheckVerifregAgainstMiners(acc *builtin.MessageAccumulator, verifregSummary
 		maddr, err := address.NewIDAddress(uint64(claim.Provider))
 		acc.RequireNoError(err, "error creating ID address: %v", err)
 
-		minerSummary, ok := minerSummaries[maddr]
+		_, ok := minerSummaries[maddr]
```

Comment:
Claims still exist after sector termination which is where all of the failures observed on mainnet traced back to.

More concerning is that claims still exist after expiration until cleanup happens.  Once simple qap FIL+ sectors start expiring this message is going to become very spammy.

Reply:
To better assert this we could change miner summaries to find terminated sectors which has its own problems as termination information can be lost today.  We could also look at the writeonly sectors bitfield for a weaker condition but that could hide problems as no sector need to be associated with it.